src/helper_functions.jl

```diff
@@ -1,3 +1,5 @@
+import InteractiveUtils: subtypes
```

Comment:
This must be done in a `VERSION` check for v0.6. I guess for the old version you need to do `import Base: subtypes`.

Reply:
ooops i forgot about that. thanks